src/main/java/com/bulkbuy/enterprise/dto/Order.java

```diff
@@ -9,15 +9,21 @@
 public @Data class Order {
 
     private int orderId;
-    private int quantity;
+    private int productQuantity;
     private String productDescription;
     private Date orderDate;
     private Date estimatedArrival;
     private String carrier;
     private String orderStatus;
 
     private int itemID;
-    private String customer;
-    private String customerAddress;
+    private int customerId;
+    private String customerFirstname;
+    private String customerLastname;;
+    private String customerStreet;
+    private String customerCity;
+    private String customerState;
+    private String customerZip;
+
```

Comment:
Are these new attributes used anywhere?  I don't see them used anywhere.  If not, this is incomplete functionality, which should not be merged to main.  Incomplete functionality is technical debt.